scripts/generate-entry-i10n.js

```diff
@@ -0,0 +1,187 @@
+// Get google translate credentials
+const keysEnvVar = process.env["GOOGLE_TRANSLATE_CREDENTIALS"];
+if (!keysEnvVar) {
+  throw new Error(
+    "The GOOGLE_TRANSLATE_CREDENTIALS environment variable was not found!"
+  );
+  return;
+}
+
+const { SUPPORTED_LANGUAGES } = require("./../constants.js");
+const { find } = require("lodash");
+const { db, pgp } = require("../api/helpers/db");
+
+const { Translate } = require("@google-cloud/translate").v2;
+const authKeys = JSON.parse(keysEnvVar);
+authKeys["key"] = process.env.GOOGLE_API_KEY;
+const translate = new Translate(authKeys);
+var dataLengthTotal = 0;
+
+// getThings('case');
```

Comment:
i think we should add a couple functions in this file to help with testing and roll out, and so we don't have to edit and deploy a new file everytime we want to run a test on an entry or comment things out. what do you think of this?

```
// translate all non-hidden entries
function translateAllEntries() {
  ...
}

// translate a list of entries where entries is an array of objects [{ type: "case", id: 1 }]
function translateListOfEntries(entries) {
  entries.forEach(entry => {
    getThingById(entry.type, entry.id);
  });
}

function countCharactersForListOfEntries() {
  ...
}

function countCharactersForAllEntries() {
  ...
}

module.exports = { translateListOfEntries, translateAll, countCharactersForListOfEntries, countCharactersForAllEntries };

```

```
// cli usage

$ heroku run node  --app participedia-i18n-staging
// Running node on ⬢ participedia-i18n-staging... up, run.9213 (Hobby)
> 

> require("generate_entry_i10n.js");

> const entries = [
  { type: "case", id: 3193 },
  { type: "organization", id: 4560 }
];

> translateListOfEntries(entries);
```

Reply:
@ascott I normally run the script on my computer. Since I'm the one who would be taking care of the translation script. We won't have to re-deploy every time we want to test. 
I think this isn't a good use of time. Also, changing a code this late, right before we want to deploy isn't a good idea.
It also would likely to be run once.

Reply:
@paninee i started on this and will push my progress to share with you.

Reply:
here is what i am proposing. i'm really only changing the interface. https://github.com/participedia/api/pull/876/files#diff-1d6197377a3babe9525e11746ba8999d